pages/organizations/College-based.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>College-Based</title>
+
+    <!--Custom CSS-->
+    <link rel="stylesheet" href="/USC-Infosite/pages/styles/global.css">
+    <link rel="stylesheet" href="/USC-Infosite/pages/styles/university-based.css">
+    <link rel="stylesheet" href="/USC-Infosite/pages/styles/status_code/status_code.css">
+
+</head>
+<body>
+    <header>
+        <div class="header__container">
+            <div class="header__banner">
+                <a href="/"><img src="/USC-Infosite/pages/assets/logos/USC-logo.png" alt="USC-logo" class="header__logo" />
+                    <div class="header__text">
+                        <h1>West Visayas State University</h1>
+                        <h2>University Student Council</h2>
+                    </div>
+                </a>
+            </div>
+            <div class="header__btns">
+                <div class="header__menu">
+                    <a href="/browse" data-js="menu_btn">
+                        <p>Menu</p>
+                        <img alt="" src="/USC-Infosite/pages/assets/icons/chevron_down.svg" data-js="menu_btn_icon" />
+                    </a>
+                </div>
+                <div class="header__search">
+                    <a href="/search" data-js="search_btn">
+                        <p class="sr_only">Search</p>
+                        <img alt="Search" src="/USC-Infosite/pages/assets/icons/magnifier.svg" />
+                    </a>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <!--College-Based Organizations Contents-->
+    <div class="title">College-Based Organizations</div>
+        <div class="title-about">Listed below are the different colleges under the university.<br> Choose one or the other to see the organizations under it.</div>
+        <div class="universities-container">
+            <div class="Organization-1">
+                <img class="org-logo" src="/USC-Infosite/pages/assets/logos/CBM-logo.png">
+                <div class="org-name"><p id="fd-name">College of Business and Management</p></div>
+                <img class="arrow" src="/USC-Infosite/pages/assets/icons/arrow-right.svg"> 
+            </div>
+            <div class="Organization">
+                <img class="org-logo" src="/USC-Infosite/pages/assets/logos/PESCAR-logo.png">
```

Comment:
Good thing you used the `<img>` tag. 

I was expecting though that the element with `.Organization` is an anchor tag (`<a>`). Each organization is its own link and by wrapping the rest of the elements under it, the clickable area is increased. That way, you are not limited to clicking the organization name but also the logo or the white space between it.

```
<a class="Organization" href="[organization-link-here]">
    <img...
</a>
```